package.json

```diff
@@ -0,0 +1,65 @@
+{
+  "name": "@google/gapic-generator-typescript",
+  "version": "0.0.1",
+  "description": "**This is not an official Google product.**",
+  "engines": {
+    "node": ">=8.0.0"
+  },
+  "files": [
+    "build/src",
+    "pbjs-genfiles"
+  ],
+  "bin": {
+    "protoc-gen-typescript_gapic": "build/src/cli.js"
+  },
+  "scripts": {
+    "test": "nyc --reporter=lcov mocha build/test && nyc report",
```

Comment:
Let's not do the . `nyc report` as part of . `npm test` and save that for the `codecov` command. @bcoe to verify that's correct 🤷‍♂

Reply:
`nyc --reporter=lcov --reporter=text mocha build/test` will do what you want, outputting to `coverage` with an `lcov` report and showing information on the terminal.

I recommend grabbing the `.nycrc` from one of our other repos however, because it has settings that our kokoro ci/cd expects.

Reply:
Done! As for making it kokoro-compatible, we'll do that when we move to kokoro (we're using CirlceCI for development).